test/e2e/storage/testsuites/provisioning.go

```diff
@@ -1139,6 +1371,55 @@ func preparePVCDataSourceForProvisioning(
 	return dataSourceRef
 }
 
+// findVolumeMountPath looks for a claim name inside a pod and returns an absolute path of its volume mount point.
```

Comment:
We probably don't need to copy this code from open [PR](https://github.com/kubernetes/kubernetes/pull/110304). I believe it could be merged soon as it was only waiting for getting fixed version of the driver into [CI](https://github.com/kubernetes/kubernetes/pull/114306) which is now passing. @mattcary what do you think?

Reply:
This PR depends on findVolumeMountPath existing as both ROX tests use it. What is the ETA on that PR merging? I'd be happy to wait for it to merge in if it's within this month (Jan 2023), but beyond that I think it needs to be merged in so we can get this use case protected.

Reply:
Agreed, unfortunately the ETA is not fully up to me. @mattcary do you see the PRs I mentioned getting merged any time soon?

Reply:
Ah, sorry, I missed the comment  on #110304.

I think it will be most expedient on that PR to just update the hostpath driver, then that PR can be submitted. I can then make a PR that updates the PD CSI version that the e2e tests use & change  InitGcePDCSIDriver.

That should unblock this PR I think as there's no GCE PD dependency?

Reply:
Yes, please, go with #2. Thanks!

Reply:
(I just created #114955 which goes into more detail of why I think #2 is correct :-)

Reply:
@mattcary Done. I had to also add the recently added [anti-capability](https://github.com/kubernetes/kubernetes/pull/110304/commits/c78bb5a43b5e4465bfdc2d6fe56f318743207c34#diff-b3ae4c017c06f067fbe5d45cd922c508eccfa25195011970a9426a35d2115cc7R826) to pass the CI.

Reply:
Looks like go/k8spr/110304 needs another lgtm after some changes/squashing - @mattcary do you have time to look? We're waiting on that to go in before this one.

Reply:
Gentle ping @mattcary